styles.css

```diff
@@ -0,0 +1,517 @@
+}
+
+.img-two {
+  background: linear-gradient(180.45deg, rgba(38, 38, 38, 0) 0.75%, rgba(38, 38, 38, 0.9) 61.94%), url("./images/portfolio/card-two.svg");
+}
+
+.img-three {
+  background-image: url("./images/portfolio/card-three.svg");
+}
+
+.img-four {
+  background-image: url("./images/portfolio/card-four.svg");
+}
+
+.img-five {
+  background-image: url("./images/portfolio/card-five.svg");
+}
+
+.img-six {
+  background-image: url("./images/portfolio/card-six.svg");
+}
+
+.img-seven {
+  background-image: url("./images/portfolio/card-seven.svg");
+}
+
+.card-hedaing {
+  width: 327px;
+  height: 36px;
+  margin: 24px 24px 12px 24px;
+  text-align: center;
+  color: #091e42;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  font-family: 'Crete Round', serif;
+}
+
+.card-heading > p {
+  width: 327px;
+  height: 147px;
+  text-align: center;
+  color: #091e42;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+  margin-top: 12px 12px;
+}
+
+.skills {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 327px;
+}
+
+.skills input[type=button] {
+  background: linear-gradient(0deg, #fff, #fff), linear-gradient(0deg, #8993a4, #8993a4);
+  padding: 10px 12px 10px 12px;
+  color: #091e42;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 20px;
+  font-family: 'Inter', serif;
+}
+
+.projects input[type=button] {
+  height: 48px;
+  width: 124px;
+  left: 0;
+  top: 259px;
+  border-radius: 0;
+  padding: 12px;
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  font-size: 17px;
+  font-weight: bold;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+}
+
+.card-heading-white {
+  margin: 60px 19px 12px 13px;
+  width: 295px;
+  height: 81px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.card-heading-white + p {
+  margin: 0 13px 12px 13px;
+  width: 301px;
+  height: 99px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.skills-details {
+  display: flex;
+  flex-direction: row;
+  max-width: 250px;
+}
+
+.skills-details input[type=button] {
+  padding: 10px 12px;
+  margin: 12px 2px 22px 13px;
+  height: 40px;
+  background: rgba(255, 255, 255, 0.24);
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.projects-details input[type=button] {
+  margin: 0;
+  height: 48px;
+  width: 327px;
+  border-radius: 0;
+  padding: 12px;
+  background: #ff6b00;
+  border: none;
+  color: #fff;
+  font-size: 17px;
+  font-weight: bold;
+  line-height: 24px;
+  font-family: 'Inter', serif;
+}
+
+/* About me Section */
+.about-section {
+  margin-top: 120px;
```

Comment:
- [ ] kindly consider removing this margin to reduce the spacing between these two sections this will make your project look precisely as the provided design and by that you will be meeting the design requirements

Reply:
Thank you very much @divinecharlotte!